infra/aws/terraform/prow-build-cluster/resources/node-problem-detector/node-problem-detector.yaml

```diff
@@ -37,6 +37,8 @@ spec:
                     operator: In
                     values:
                       - linux
+      nodeSelector:
```

Comment:
Is this deployment missing tolerations?

Reply:
@upodroid Great catch, I dropped changes for this file. PTAL again